pants.toml

```diff
@@ -15,14 +18,20 @@ backend_packages = [
                  "sendwave.pants_node",
                  "sendwave.pants_docker"
                  ]
+
 [source]
-root_patterns = ["/pants_plugins", "/test_webpack"]
+root_patterns = ["/pants_plugins", "/", "/test_webpack"]
```

Comment:
this root was required because we reference 3rd party requirements using the `//:` syntax which requires explicitly adding the build root to the root_patterns